src/command.c

```diff
@@ -577,7 +586,7 @@ void dv_run_conv_command(struct dmp_cmb *cmb, void *bar_logical)
 }
 
 static int dv_convert_fc_v0(struct device *dev, struct dmp_cmb *cmb,
-			    struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
+		struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket